pkl-core/src/main/java/org/pkl/core/runtime/StackTraceRenderer.java

```diff
@@ -15,15 +15,23 @@
  */
 package org.pkl.core.runtime;
 
+import static org.fusesource.jansi.Ansi.ansi;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.function.Function;
+import org.fusesource.jansi.Ansi;
+import org.fusesource.jansi.Ansi.Color;
 import org.pkl.core.StackFrame;
 import org.pkl.core.util.Nullable;
 
 public final class StackTraceRenderer {
   private final Function<StackFrame, StackFrame> frameTransformer;
 
+  private static final Ansi.Color frameColor = Color.YELLOW;
+  private static final Ansi.Color lineNumColor = Color.BLUE;
+  private static final Ansi.Color repColor = Color.MAGENTA;
```

Comment:
Happy to see you're choosing colours that don't wobble to us deuteranomalous folks; that said... should these be configurable? More generally, I feel it's probably wise to put all the colour definitions in one place.

Besides that: I'm not _entirely_ certain what `rep` is the abbreviation of... guessing here;
```suggestion
  private static final Ansi.Color frameColor = Color.YELLOW;
  private static final Ansi.Color lineNumColor = Color.BLUE;
  private static final Ansi.Color repetitionColor = Color.MAGENTA;
```

Reply:
Yeah `rep` is `repetition`. The colour choice being deuteranomalous friendly was more by accident than design. Was trying to stick to the basic colours that have very wide support, rather than trying to get fancy. Also means that colours will appear slightly different in different terminals, and I imagine that most terminals support customising the precise RGB values of each colour.

Wouldn't recommend making these configurable right now. The formatting API doesn't really lend itself to making that easy as the final colour is determined by the base colour here, but also the exact method used to apply the colour e.g. `fgBright()` vs `fg`. If we were serious about allowing colour configuration, then I think we should also look at reworking how these outputs are generated, and moving this formatting out of `pkl-core` and into `pkl-cli-common`, and adding extra info to the `PklException` so that those layers can access the underlying failure and render a completely different output based on terminal capability and user config.

Reply:
Yeah `rep` is `repetition`. The colour choice being deuteranomalous friendly was more by accident than design. Was trying to stick to the basic colours that have very wide support, rather than trying to get fancy. Also means that colours will appear slightly different in different terminals, and I imagine that most terminals support customising the precise RGB values of each colour.

Wouldn't recommend making these configurable right now. The formatting API doesn't really lend itself to making that easy as the final colour is determined by the base colour here, but also the exact method used to apply the colour e.g. `fgBright()` vs `fg`. If we were serious about allowing colour configuration, then I think we should also look at reworking how these outputs are generated, and moving this formatting out of `pkl-core` and into `pkl-cli-common`, and adding extra info to the `PklException` so that those layers can access the underlying failure and render a completely different output based on terminal capability and user config.

Reply:
Yeah `rep` is `repetition`. The colour choice being deuteranomalous friendly was more by accident than design. Was trying to stick to the basic colours that have very wide support, rather than trying to get fancy. Also means that colours will appear slightly different in different terminals, and I imagine that most terminals support customising the precise RGB values of each colour.

Wouldn't recommend making these configurable right now. The formatting API doesn't really lend itself to making that easy as the final colour is determined by the base colour here, but also the exact method used to apply the colour e.g. `fgBright()` vs `fg`. If we were serious about allowing colour configuration, then I think we should also look at reworking how these outputs are generated, and moving this formatting out of `pkl-core` and into `pkl-cli-common`, and adding extra info to the `PklException` so that those layers can access the underlying failure and render a completely different output based on terminal capability and user config.

Reply:
Yeah `rep` is `repetition`. The colour choice being deuteranomalous friendly was more by accident than design. Was trying to stick to the basic colours that have very wide support, rather than trying to get fancy. Also means that colours will appear slightly different in different terminals, and I imagine that most terminals support customising the precise RGB values of each colour.

Wouldn't recommend making these configurable right now. The formatting API doesn't really lend itself to making that easy as the final colour is determined by the base colour here, but also the exact method used to apply the colour e.g. `fgBright()` vs `fg`. If we were serious about allowing colour configuration, then I think we should also look at reworking how these outputs are generated, and moving this formatting out of `pkl-core` and into `pkl-cli-common`, and adding extra info to the `PklException` so that those layers can access the underlying failure and render a completely different output based on terminal capability and user config.